xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -716,13 +792,22 @@ The JSON string contains invalid UTF-8 bytes or invalid UTF-16 surrogates.</exce
       <Docs>
         <summary>Parses the current JSON token value from the source as a <see cref="T:System.UInt16" />.</summary>
         <returns>The UTF-8 encoded token value parsed to a <see cref="T:System.UInt16" />.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
```

Comment:
Some of the overloads had this text already (like GetDecimal), along with a link to `See also: TokenType` (which were added in https://github.com/dotnet/dotnet-api-docs/pull/1880).
So, I added it to all the other Get* APIs for consistency.
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.utf8jsonreader.getdecimal?view=netcore-3.0

![image](https://user-images.githubusercontent.com/6527137/64501669-24b08480-d277-11e9-91c5-31590e181789.png)

Reply:
I left you a comment regarding this. I think the exception information should contain this text, not the remark.